src/schema/feeds.ts

```diff
@@ -230,6 +230,46 @@ export const typeDefs = /* GraphQL */ `
       supportedTypes: [String!]
     ): PostConnection! @auth
 
+    """
+    Get feed preview
+    """
+    feedPreview(
+      """
+      Time the pagination started to ignore new items
+      """
+      now: DateTime
+
+      """
+      Paginate after opaque cursor
+      """
+      after: String
+
+      """
+      Paginate first
+      """
+      first: Int
+
+      """
+      Ranking criteria for the feed
+      """
+      ranking: Ranking = POPULARITY
+
+      """
+      Return only unread posts
+      """
+      unreadOnly: Boolean = false
+
+      """
+      Version of the feed algorithm
+      """
+      version: Int = 1
```

Comment:
We need to use a different version than the regular feed. We may want to experiment differently. So we can refer to it internally and product wise as onboarding feed version. We will have to create dedicated gb flag for that. `onboarding_feed`

Reply:
@idoshamun what would be the version, since others are incremental?

Reply:
In actual resolver we do not look at this field at all we always use new onboarding generator.

Reply:
We have two options: either support experimenting right now (from schema pov) or delay it. I don't have strong opinion

Reply:
Removed it for now.